test/selenium/lib/testPrepperSelenium.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
Same than previous comment, using what No-Jun implemented for integration but with less options not needed so far for the UI tests